DirectXTex/DirectXTexMipmaps.cpp

```diff
@@ -2521,6 +2521,129 @@ namespace
 // Entry-points
 //=====================================================================================
 
+float clamp(float value, float min, float max)
+{
+    return value < min ? min : value > max ? max : value;
+}
+
+//-------------------------------------------------------------------------------------
+// Alpha to coverage, for mip generation
+//-------------------------------------------------------------------------------------
+_Use_decl_annotations_
```

Comment:
This only works if you have a SAL annotated prototype. Since it's a locally declared helper, you need to SAL annotate it here.

You can validate this by using ``/analyze``.